src/lib/InstantSearch.ts

```diff
@@ -184,6 +185,18 @@ See: https://www.algolia.com/doc/guides/building-search-ui/going-further/backend
       );
     }
 
+    warning(
+      Boolean(insightsClient) || !hasDetectedInsightsClient(),
+      `InstantSearch detected the insights client in the global scope. 
```

Comment:
We capitalize "Insights" [everywhere in the doc](https://www.algolia.com/doc/guides/getting-insights-and-analytics/search-analytics/click-through-and-conversions/in-depth/cookie-strategy/#identifying-users-with-a-cookie-and-a-user-token).

```suggestion
      `InstantSearch detected the Insights client in the global scope.
```